.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

Comment:
Also, unlike our current tools, 

# MARKDOWN WORKS!

This means you can do things like [link to other parts of the code](https://github.com/neilospricey/emoji-search/blob/master/Dockerfile) quite easily